crates/node_types/lightclient/src/lightclient.rs

```diff
@@ -278,51 +282,69 @@ impl LightClient {
         None
     }
 
-    async fn process_epoch(&self, height: u64) -> Result<()> {
-        info!("processing epoch at height {}", height);
-        self.event_publisher.send(LightClientEvent::EpochVerificationStarted { height });
+    async fn poop_epoch(&self, finalized_epoch: &FinalizedEpoch) -> Result<()> {
```

Comment:
Please think of a better name, probably `process_epoch` is best, and we should rename `process_epoch` to `process_height`